src/components/Disturber.tsx

```diff
@@ -39,6 +40,9 @@ interface DataItem {
 // eslint-disable-next-line complexity
 export const Disturber = ({ navigation, publicJsonFile }: Props) => {
   const [isVisible, setIsVisible] = useState(false);
+  const { globalSettings } = useContext(SettingsContext);
+  const { sections = {} } = globalSettings;
+  const { showDisturberHiddenButton = true } = sections;
```

Comment:
please use a more readable name, like you would speak it, for example: `showButtonToHideDisturber`
quick translation to German with deepl:
<img width="1131" alt="image" src="https://github.com/smart-village-solutions/smart-village-app-app/assets/1942953/b97b5f46-a35a-4648-a46f-9d8b6bab25ef">

Reply:
maybe `close` would be better than `hide` as the other code uses the word close.

Reply:
Yes, I will fix it. You're right, I'll use `close` to make the code complete.